### PR TITLE
Boolean logic update

### PR DESCRIPTION
**Pull request by SJ-Nosrat**
### Update - `concepts.wip/boolean-logic`

- Add blurb to `config.json`
- Add about to `about.md`
- Add introduction to `introduction.md`
- Add links to `links.json`

### CODE REVIEW

File: concepts.wip/boolean-logic/.meta/config.json [hunk 1]
```diff
@@ -1,5 +1,5 @@
 {
-  "blurb": "TODO: add blurb for boolean-logic concept",
-  "authors": [],
+  "blurb": "Boolean logic forms part of Boolean algebra, and involves combining logical statemets using logical connectives, which evaluates to either true or false.",
```
Comment (SaschaMann) [inline concepts.wip/boolean-logic/.meta/config.json:2]: Many people find maths terms scary, so perhaps we can remove it from here:

```suggestion
  "blurb": "Boolean logic involves combining logical statements using logical connectives, which evaluate to either true or false.",
```

File: concepts.wip/boolean-logic/about.md [hunk 2]
```diff
@@ -1,3 +1,134 @@
 # About
 
-<!-- TODO: add information on boolean-logic concept -->
+Boolean logic is a branch of Boolean algebra, where boolean expressions are combined using logical operators, in Julia called [boolean operators](https://docs.julialang.org/en/v1/manual/mathematical-operations/#Boolean-Operators), to form logical statements that evaluates to either `true` or `false`.
```
Comment (SaschaMann) [inline concepts.wip/boolean-logic/about.md:3]:
```suggestion
Boolean logic is a branch of Boolean algebra, where boolean expressions are combined using logical operators, in Julia called [boolean operators](https://docs.julialang.org/en/v1/manual/mathematical-operations/#Boolean-Operators), to form logical statements that evaluate to either `true` or `false`.
```

**Review by SaschaMann — APPROVED** [review]
Great work, some minor suggestions :)

File: concepts.wip/boolean-logic/.meta/config.json (hunk 1 above)
Comment (SJ-Nosrat) [reply concepts.wip/boolean-logic/.meta/config.json:2]: Done. Thanks for the suggestion.

File: concepts.wip/boolean-logic/about.md (hunk 2 above)
Comment (SJ-Nosrat) [reply concepts.wip/boolean-logic/about.md]: Done. Thanks for the suggestion.

**Pull request merged**